src/main/java/org/zalando/pazuzu/feature/FeaturesMetaResource.java

```diff
@@ -47,10 +46,6 @@ public FeaturesMetaResource(FeatureService featureService) {
             }
         }
         Set<Feature> featureSet = featureService.loadFeatures(Arrays.stream(featureNames).collect(Collectors.toList()));
-        if (sorting != null && sorting.equals(TOPOLOGICAL_SORT)) {
```

Comment:
This sorting is not required by the new api so I removed it. Maybe we add a sorting capability to the new api but this will be in a separate issue